components/Common/Breadcrumbs/BreadcrumbRoot/index.tsx

```diff
@@ -0,0 +1,31 @@
+import HomeIcon from '@heroicons/react/24/outline/HomeIcon';
+import type { FC, PropsWithChildren } from 'react';
+
+import BreadcrumbItem from '@/components/Common/Breadcrumbs/BreadcrumbItem';
+import LocalizedLink from '@/components/LocalizedLink';
+
+import styles from './index.module.css';
+
+type BreadcrumbRootProps = {
+  hideHome?: boolean;
+};
+
+const BreadcrumbRoot: FC<PropsWithChildren<BreadcrumbRootProps>> = ({
+  hideHome = false,
```

Comment:
I usually prefer non-negated logic, like `showHome`

Reply:
Thanks for commenting on this. Generally I tend to follow the prop naming convention by [MUI](https://mui.com/material-ui/guides/api/#prop-naming). I assumed that the Home button will be shown by default, hence we can do `hideHome` instead of `showHome={false}`

Reply:
I see, I thought for one second that this was the BreadcrumbItem component. Yea negating here makes sense